src/components/MatchModal/index.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { func } from 'prop-types';
+import { View, Text, Image, TouchableOpacity } from 'react-native';
+
+import Modal from 'components/common/Modal';
+import strings from 'locale';
+import { MATCH_SHAPE } from 'constants/shapes';
+import ovals from 'assets/images/match.png';
+import styles from './styles';
+
+const {
+  MATCH: { modalTitle, message, skip },
+} = strings;
+
+const MatchModal = ({ match, onHide }) => (
+  <Modal>
+    <Image source={ovals} />
+    <Text style={styles.title}>{modalTitle}</Text>
+    <Text style={styles.message}>{message}</Text>
+    <View style={styles.matchedUser}>
+      {match.avatar.smallThumbUrl && (
+        <Image source={{ uri: match.avatar.smallThumbUrl }} style={styles.avatar} />
+      )}
```

Comment:
you could use a default profile picture when the user doesn't have one.

Reply:
I agree! Thanks! But there is no such one in the designs...

Reply:
actually there is, in the Edit Profile Screen: 

![image](https://user-images.githubusercontent.com/8755889/80639342-3fb74b00-8a38-11ea-8b48-3ae860fed728.png)

Reply:
Oh, great! Thank you!